profile/README.md

```diff
@@ -0,0 +1,90 @@
+
+If you write proprietary software and want better integration with these crates by all means raise
+issues on the respective repository. We are interested in all users and your usecase may help us
+improve the libraries, however all development is done in the open and will stay that way.
+
+[0]  (Please note `internals` will explicitly never stabalize.)
+
+## Other Crates
+
+Other crates/repositories hosted here and developed and maintained by the community.
+
+These crates are developed and maintained by a subset of the `rust-bitcoin` devs or other members of
+the community. We are fairly open about hosting any bitcoin-related crates in this org, provided
+that they are actively maintained and:
+
+- Repository contains Bitcoin specific crates (implies Bitcoin Only and not general purpose crates)
+- Repository has at least minimal CI
+- Author has made at least some effort to adhere to practices and standards maintained in the core
+  crates (defined above).
+
+These crates may not be reviewed or maintained by the primary `rust-bitcoin` maintainers. Keep this in
+mind when choosing to use them as dependencies. And as with any dependencies, you should review new
+versions rather than blindly upgrading.
+
+Name | Crate | Repository
+---|---|---
+miniscript | [miniscript](https://crates.io/crates/miniscript) | [rust-miniscript](https://github.com/rust-bitcoin/rust-miniscript)
+bitcoind | [bitcoind](https://crates.io/crates/bitcoind) | [bitcoind](https://github.com/rust-bitcoin/bitcoind)
+bitcoincore-rpc | [bitcoincore-rpc](https://crates.io/crates/bitcoincore-rpc) | [rust-bitcoincore-rpc](https://github.com/rust-bitcoin/rust-bitcoincore-rpc)
+bip39 | [bip39](https://crates.io/crates/bip39) | [rust-bip39](https://github.com/rust-bitcoin/rust-bip39)
+bip47 | [bip47](https://crates.io/crates/bip47) | [rust-bip47](https://github.com/rust-bitcoin/rust-bip47)
+murmel | [murmel](https://crates.io/crates/murmel) | [murmel](https://github.com/rust-bitcoin/murmel)
+hammersbald | [hammersbald](https://crates.io/crates/hammersbald) | [hammersbald](https://github.com/rust-bitcoin/hammersbald)
+bitcoin-bech32 | [bitcoin-bech32](https://crates.io/crates/bitcoin-bech32) | [rust-bech32-bitcoin](https://github.com/rust-bitcoin/rust-bech32-bitcoin)
+
+## Non-crate repositories
+
+Repositories that are not crates (libraries).
+
+Name | Repository | Description
+---|---|---
+maintainer-tools | [rust-bitcoin-maintainer-tools](https://github.com/rust-bitcoin/rust-bitcoin-maintainer-tools) | Scripts, test vectors, and other things used by or across multiple repositories in the `rust-bitcoin` ecosystem.
+website | [rust-bitcoin.github.io](https://github.com/rust-bitcoin/rust-bitcoin.github.io) | Source for the org website https://rust-bitcoin.org
+workshop | [workshop](https://github.com/rust-bitcoin/) | Conference presentation material
+.github | [.github](https://github.com/rust-bitcoin/) | Hosts this readme file
+
+## Archived (moved or unmaintained)
```

Comment:
```suggestion
## Archived (moved or unmaintained) crates
```

Also we should add a brief explainer, as all preceding H2, before the table.

Reply:
Added "These crates have either been moved to another location or are no longer actively maintained."